src/config/dynamic-theme-fixes.config

```diff
@@ -746,8 +746,8 @@ INVERT
 .sheet-name-container
 .sheet-attributes-container
 .sheet-attr-container button
-.sheet-row
-.sheet-header
+sheet-row
```

Comment:
These 2 entries can be removed entirely than your good to go to be merged.

Reply:
I'm sorry but I am not sure I follow the difference?

Reply:
I mean, the selectors don't serve an purpose now so if you remove them it's better than heaving broken CSS in the fixes config.

Reply:
Oh my goodness, I think I see now - I was essentially removing them got it - do I need to do anything to accept your alteration or anything?

Reply:
> do I need to do anything to accept your alteration or anything?

Give me a moment